src/lightning_app/core/api.py

```diff
@@ -345,6 +348,13 @@ async def get_status() -> AppStatus:
     return app_status
 
 
+@fastapi_service.get("/api/v1/annotations", response_class=JSONResponse)
+async def get_annotations() -> Union[List, Dict]:
+    """Get the annotations associated with this app."""
+    global app_annotations
+    return app_annotations or []
```

Comment:
I wonder if it will make it cleaner/more maintainable if we will load annotations on request? This will allow us to avoid using the global variable. Annotations will be requested only once on page load anyway.

Reply:
As discussed offline this may be the eventual direction but leaving it as is for now to minimise any performance hit associated with the endpoint